aioredisqueue/queue.py

```diff
@@ -34,13 +40,22 @@ def __init__(self, redis, key_prefix='aioredisqueue_', *,
             'fifo': fetching_fifo_key,
             'payload': payloads_hash_key,
             'ack': ack_hash_key,
+            'last_requeue': last_requeue_key,
         }
 
         self._redis = redis
         self._loop = loop
         self._task_class = task_class
         self._lua_sha = lua_sha if lua_sha is not None else {}
         self._locks = {}
+        self._requeue_interval = requeue_interval
+        self._is_stopped = False
+
+        if self._requeue_interval != 0:
+            self._regular_requeue_task = \
+            self._loop.create_task(self._requeue_regularly())
```

Comment:
Hmm. 
Creating a task in constructor doesn't feel right. Especially if we have to call `.stop()` afterwords (we did't `.start()` or `.run()` anything, right?).

Reply:
When we call `__init__` it is not guaranteed that loop is activated and `redis` has connection (although it wouldn't be a very popular use case).
So we have three options:
1) start periodic task only when some other method called,  `requeue_interval != 0` and corresponding `future` is `None`
2) Wrap everything into `__aenter__` and `__aexit__` so to use an instance one will have to call create it with `async with`. It is unclear whether we should or should not support calls outside `async with` block.
3) Provide `__aenter__` and `__aexit__` only for periodic task. This is a little bit more flexible option for user as periodic task can be stopped and started multiple times. But it is kind of harder to implement: We need to provide separate class (asynchronous context manager) for this particular case.

Reply:
Probably the easiest way for now is to provide `run_periodic()` method that will have `requeue_interval` argument. Corresponding `stop_periodic()` will have obvious semantics. Your more general `.stop()` method will have to go to a different pull request (or to `/dev/null`)